example/src/app.controller.ts

```diff
@@ -14,4 +16,9 @@ export class AppController {
     this.monitoringService.incrementRequestCounter();
     return this.appService.getHello();
   }
+
+  @Get('/route')
+  randomRoute(): string {
+    return 'Hello from random route!';
```

Comment:
Add a random sleep between 20ms to 2500ms so that it shows in the histogram. Do this in such a way that you can verify in the e2e testh. Possibly create a `GET` param called request delay.

Reply:
done.